lang/java/reef-runtime-azbatch/pom.xml

```diff
@@ -42,6 +42,11 @@ under the License.
             <artifactId>reef-common</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
```

Comment:
reef-runtime-azbatch should not have dependency on reef-runtime-hdinsight. We probably want to move AzureUploader to common package or util package.

Reply:
Like I stated in the comment of the PR, we need to see if using Batch Application packages is an option. If it is, we won't even need AzureUploader, so it doesn't make sense to refactor now.

Reply:
somehow I missed 3rd notes... probably becasue of you say two things to note. :)